lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -68,13 +73,53 @@ def main(argv=None):
                              'relative to this cycletime. '
                              'This cycletime is in the format '
                              'YYYYMMDDTHHMMZ.')
-    # Filepaths for historic and truth data.
-    parser.add_argument('historic_filepath', metavar='HISTORIC_FILEPATH',
-                        help='A path to an input NetCDF file containing the '
-                             'historic forecast(s) used for calibration.')
-    parser.add_argument('truth_filepath', metavar='TRUTH_FILEPATH',
-                        help='A path to an input NetCDF file containing the '
-                             'historic truth analyses used for calibration.')
+
+    # Historic forecast and truth filepaths
+    parser.add_argument(
+        '--historic_filepath', metavar='HISTORIC_FILEPATH', nargs='+',
+        help='A path to an input NetCDF file containing the '
+             'historic forecast(s) used for calibration. '
+             'This must be supplied with an associated truth filepath. '
+             'Specification of either the combined_filepath, '
+             'historic_forecast_identifier or the truth_identifier is '
+             'invalid with this argument.')
+    parser.add_argument(
+        '--truth_filepath', metavar='TRUTH_FILEPATH', nargs='+',
+        help='A path to an input NetCDF file containing the '
+             'historic truth analyses used for calibration. '
+             'This must be supplied with an associated historic filepath. '
+             'Specification of either the combined_filepath, '
+             'historic_forecast_identifier or the truth_identifier is '
+             'invalid with this argument.')
+
+    # Input filepaths
+    parser.add_argument(
+        '--combined_filepath', metavar='COMBINED_FILEPATH', nargs='+',
+        help='The path to the input NetCDF files containing '
+             'both the historic forecast(s) and truth '
+             'analyses used for calibration. '
+             'This must be supplied with both the '
+             'historic_forecast_identifier and the truth_identifier. '
+             'Specification of either the historic_filepath or the '
+             'truth_filepath is invalid with this argument.')
+    parser.add_argument(
+        "--historic_forecast_identifier",
+        metavar='HISTORIC_FORECAST_IDENTIFIER',
+        help='The path to a json file containing metadata '
+             'information that defines the historic forecast. '
+             'This must be supplied with both the combined_filepath and the '
+             'truth_identifier. Specification of either the historic_filepath'
+             'or the truth_filepath is invalid with this argument.')
+    parser.add_argument(
+        "--truth_identifier", metavar='TRUTH_IDENTIFIER',
+        help='The path to a json file containing metadata '
```

Comment:
Could link to the IMPROVER function where the format of this json file is defined? Is that what we've done for this kind of thing before?

Reply:
I've added a link to the CLI help.